guides/doc-Provisioning_Guide/topics/Cloud-GCE.adoc

```diff
@@ -42,15 +40,38 @@ To add a connection, complete the following steps:
 . From the *Zone* list, select the GCE zone to use.
 . Click *Submit*.
 
+.For CLI Users
+
+. In GCE, generate a service account key in JSON format and upload this file to the `/usr/share/foreman/` directory on {ProjectServer}.
+
+. On {ProjectServer}, configure permissions for the service account key to ensure that the file is readable by the `foreman` user:
++
+[options="nowrap" subs="+quotes"]
+----
+# chown foreman /usr/share/foreman/_gce_key_.json
+# chmod 0600 /usr/share/foreman/_gce_key_.json
+# restorecon -vv /usr/share/foreman/_gce_key_.json
```

Comment:
Sergei, several times in different reviews, you have made me separate several commands into individual steps. Can you do the same here?